.prettierrc

```
@@ -0,0 +1,11 @@
+{
+  "singleQuote": true,
+  "printWidth": 100,
```

У нас везде длинна строки `120`
```suggestion
  "printWidth": 120,
```